Documentation/quickstart/quickstart.rst

```diff
@@ -0,0 +1,107 @@
+.. include:: /substitutions.rst
+.. _quickstart:
+
+Quickstart
+==========
+
+Here's the quick version of getting things going. This is a bare-bones outline for experienced developers– if it's
+going too quickly, dive into the other sections of the Companion. This Quickstart guide assumes you're on a Linux
+computer, you're using an ARM processor on your embedded board, and you're familiar with using the command line.
+
+#. Install a Cross-Compiler Toolchain
+
+   With NuttX, you compile the operating system and your application on your desktop or laptop computer, then install the
+   binary file on your embedded computer. This guide assumes your computer is an
+   `ARM <https://en.wikipedia.org/wiki/ARM_architecture>`_ CPU. If it isn't, you'll need a different tool chain.
+
+   Download the right flavor of the
+   `ARM Embedded Gnu Toolchain <https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm>`_
+   for your embedded processor's CPU.
+
+   Unpack it into ``/opt/gcc`` and add the bin directory to your path. For instance:
+
+   .. code-block:: bash
+
+      $ sudo mkdir /opt/gcc
+      $ sudo chgrp -R users /opt/gcc
+      $ cd /opt/gcc
+      $ wget https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2019q4/gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2?revision=108bd959-44bd-4619-9c19-26187abf5225&la=en&hash=E788CE92E5DFD64B2A8C246BBA91A249CB8E2D2D
+      $ tar xf gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2
+      $ # add the toolchain bin/ dir to your path...
+      $ # you can edit your shell's rc files if you don't use bash
+      $ echo "export PATH=/opt/gcc/gcc-arm-none-eabi-9-2019-q4-major/bin:$PATH" >> ~/.bashrc
+
+#. Download Apache NuttX
+
```

Comment:
```suggestion
The next step is to download NuttX main repository along the application repository. The latter is technically optional in a very minimal configurations but should be included in normal configuration since it includes the NuttX shell.
```